mne/source_estimate.py

```diff
@@ -1370,18 +1370,14 @@ def plot(self, subject=None, surface='inflated', hemi='lh',
         hemi : str, 'lh' | 'rh' | 'split' | 'both'
             The hemisphere to display. Using 'both' or 'split' requires
             PySurfer version 0.4 or above.
-        colormap : str
-            The type of colormap to use.
+        colormap : str | array
```

Comment:
@wronk sorry to torture you, whenever you specify an array type in a doc please adopt the folowing scheme, based on an example:

np.ndarray of float, shape(n_channels, n_times)